vignettes/germany-age-stratified-nowcasting.Rmd.orig

```diff
@@ -55,29 +56,24 @@ We first filter out the data that would have been available on the 1st of Septem
 ```{r}
 nat_germany_hosp <- epinowcast::germany_covid19_hosp[location == "DE"]
 
-retro_nat_germany <- enw_retrospective_data(
-  nat_germany_hosp,
-  rep_date = as.Date("2021-09-01"), ref_date = as.Date("2021-09-01") - 40
-)
+retro_nat_germany <- nat_germany_hosp |>
+  enw_filter_report_dates(latest_date = "2021-09-01") |>
+  enw_filter_reference_dates(include_days = 40)
```

Comment:
We just should be aware that by using the new pipe operator, we are expecting R.4.1 or higher.

Reply:
Yeah, good point. I've kept this to just the vignettes and readme as it seemed having nice-looking code would be pretty key to keep people on board (hence #94). It doesn't appear in any of the actual package code so I think we are okay dependencies wise.